crust.py

```diff
@@ -60,3 +69,4 @@ def main():
 
 if __name__ == '__main__':
     main()
+
```

Comment:
you know what houndci maybe I want a damn line at the end of my file because it's in the damn PEP8 standard